src/Stryker.Core/Stryker.Core/TestRunners/TestCoverageInfos.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.Extensions.Logging;
+using Stryker.Core.Logging;
+
+namespace Stryker.Core.TestRunners
+{
+    public class TestCoverageInfos
```

Comment:
As of now, the class contains no logic besides how to convert a test => mutants mapping to mutant => test.
Regarding reporting, or serializing, the best practice is to have dedicated classes for that, instead of crafting your structure to match reporting needs.

It may need refactoring, but I tend to think that this feature is still in early stage, so let's gain some experience and then the design should be obvious